library/src/scripts/events/EventsList.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * @author Adam Charron <adam.c@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import React, { Component } from "react";
+
+import { IEvent } from "@library/events/Event";
+import { eventsClasses } from "@library/events/eventStyles";
+import { Event } from "@library/events/Event";
+import classNames from "classnames";
+import { t } from "@vanilla/i18n";
+import { EventAttendance } from "@library/events/EventAttendanceDropDown";
+import { ISelectBoxItem } from "@library/forms/select/SelectBox";
+
+export interface IEventList {
+    headingLevel?: 2 | 3;
+    data: IEvent[];
+}
+
+/**
+ * Component for displaying an accessible nicely formatted time string.
+ */
+export function EventsList(props: IEventList) {
+    const classes = eventsClasses();
+    if (!props.data || props.data.length === 0) {
+        return <p className={classes.empty}>{t("This category does not have any events.")}</p>;
```

Comment:
```suggestion
        return <p className={classes.empty}>{t("There are no events.")}</p>;
```

Sorry I missed this on the first pass, but we should either change this to be more generic, or make it passable from outside. This widget may be used in anything events attach to, which could be a category, but also could be a group currently.